### DISCUSSION TITLE
Looking for where gettext was used

### DISCUSSION THREAD

**Discussion by Boojapho** [strimzi/strimzi-kafka-operator#5815] [Q&A] [answered]
In [this Dockerfile](https://github.com/strimzi/strimzi-kafka-operator/blob/main/docker-images/kafka/Dockerfile#L9), `gettext` is installed.  One of our vulnerability scanners flagged it against [this CVE](https://access.redhat.com/security/cve/CVE-2021-3114) (and 3 low CVEs).  I was trying to find where `gettext` is used and can't seem to find anything in the code.  

Can someone help point me to where it is and whether `gettext` is still required?

**Comment by scholzj** [comment, marked as answer]
`envsubst`? I think that is used in several places.

**Reply by Boojapho** [reply]
Yep that was it.  Found it in the kafka_config_generator.sh.  Thanks!
